src/adapter/src/catalog/apply.rs

```diff
@@ -188,6 +194,11 @@ impl CatalogState {
                 }
             }
         }
+
+        if update_system_config {
+            self.system_configuration.dyncfg_updates();
+        }
```

Comment:
Alternatively, we could just put the single line `self.system_configuration.dyncfg_updates();` at the end of the `apply_system_configuration_update()` method. However, on start up we'd end up updating every single parameter once per parameter (i.e. O(n^2)), which feels wasteful. This is easy enough and should be much faster.